src/main/java/org/venice/beachfront/bfapi/model/Detection.java

```diff
@@ -0,0 +1,33 @@
+package org.venice.beachfront.bfapi.model;
```

Comment:
What's the policy on file headers? Are we still using them? Several files in this PR are missing them.

Reply:
Great point. This will be a final go-to after much of the code is done shifting around.